ui-lib/src/main/res/ui/advanced_settings/values-es/strings.xml

```diff
@@ -1,9 +1,9 @@
 <resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="advanced_settings_title">Advanced Settings</string>
-    <string name="advanced_settings_recovery">Recovery Phrase</string>
-    <string name="advanced_settings_export">Export Private Data</string>
-    <string name="advanced_settings_choose_server">Choose a Server</string>
-    <string name="advanced_settings_currency_conversion">Currency Conversion</string>
-    <string name="advanced_settings_info">You will be asked to confirm on the next screen</string>
-    <string name="advanced_settings_delete_button">Delete Zashi</string>
```

Comment:
@true-jared wrote:
> "Delete Zashi" is confusing, as doing this doesn't actually delete the app. "Delete wallet/account" may be better.

What this is actually doing is deleting access to the wallet from the device. It may or may not be backed up.